### ISSUE TITLE
Acceptable Use Policy

### ISSUE THREAD

**Issue by galenp** [galenp/PROPERTYCOMPASS.ISSUES#79]
I would like us to have a new Acceptable Use Policy (AUP) accessible for Property Compass.

I think the existing www.propertycompass.com.ay/terms is dated and not necessarily applicable for the new PC.

REA has a relatively simply stated policy here https://about.realestate.com.au/acceptable-use-policy-residential/ that I think we can derive from.

**Comment by galenp** [comment]
I've updated the Add Listing screen of Property Compass as so:

![image](https://cloud.githubusercontent.com/assets/1113659/11577623/198e2fde-9a5a-11e5-987a-007f2eec2c63.png)

**Comment by anthonyaoun** [comment]
I think we can reduce this text to

It is acceptable to use this service for the bona fide listing of real property for sale and in accordance with the terms of the Acceptable Use Policy. 

Please confirm this property is for sale and you have authority to sell it.

AA

**Comment by galenp** [comment]
Did you see my initial comment in this issue.. regarding a rewrite of the PC terms. Do you think this can/should be done?

**Comment by anthonyaoun** [comment]
Yes definitely; but not at the moment. Keep the link pointing to our terms

AA

**Comment by galenp** [comment]
@anthonyaoun could you please investigate and see if a Refresh of the Acceptable Use Terms are valid for the current Property Compass
